srcpkgs/cargo-deny/template

```diff
@@ -1,23 +1,32 @@
 # Template file for 'cargo-deny'
 pkgname=cargo-deny
-version=0.12.2
+version=0.13.5
 revision=1
+_adv_commit="222ccf84ef7d733a6eac71f694a5e14b4d0d0289"
 build_style=cargo
 configure_args="--no-default-features"
 hostmakedepends="pkg-config"
 makedepends="openssl-devel"
+checkdepends="git"
```

Comment:
this probably clobbers chroot-git (#39577), can we use that instead? you'll just need to add `/usr/libexec/chroot-git` to PATH

Reply:
This probably also applies to python3-GitPython then, right?

Reply:
and probably a bunch of other things too :)

Reply:
yay. In that case, wouldn't it make more sense to fix it on a lower level anyway? Knowing that you have to get certain tools from certain other paths is increasing the barrier-to-entry for new maintainers quite a bit IMO

Reply:
yes, it does need a lower-level fix (as discussed in the issue) but we shouldn't break things in the meantime

Reply:
Agreed, which is why I adjusted this to not require `git` in `checkdepends` for now. I'll mark this as resolved.